src/DotNetWorker.Core/Hosting/WorkerOptions.cs

```diff
@@ -22,12 +23,17 @@ public class WorkerOptions
         /// </summary>
         public InputConverterCollection InputConverters { get; } = new InputConverterCollection();
 
+        /// <summary>
+        /// Gets the worker capabilities.
+        /// </summary>
+        public IDictionary<string, string> Capabilities { get; } = new Dictionary<string, string>();
```

Comment:
What do you think of having this dictionary initialize to the default set of capabilities? Will remove the need for our configure call.

Reply:
Yeah, I'm transitioning this that way. Good idea.